src/vocab.rs

```diff
@@ -34,6 +34,28 @@ impl PyVocab {
             }
         })
     }
+
+    fn ngram_indices(&self, word: &str) -> PyResult<Option<Vec<(String, usize)>>> {
+        let embeds = self.embeddings.borrow();
+        match embeds.vocab() {
+            VocabWrap::FastTextSubwordVocab(inner) => Ok(inner.ngram_indices(word)),
+            VocabWrap::FinalfusionSubwordVocab(inner) => Ok(inner.ngram_indices(word)),
+            VocabWrap::SimpleVocab(_) => Err(exceptions::ValueError::py_err(
+                "ngram-indices queries are not supported for vocabularies of type 'SimpleVocab'",
```

Comment:
"ngram-indices queries are not supported for vocabularies of type 'SimpleVocab'" -> "querying n-gram indices is not supported for this vocabulary"

The Rust type is an implementation detail.

It is currently not possible for the caller to know this. So maybe it would be nice to add a method `is_subword_vocab`. Though this can go to a separate PR.